docs/specviz/plugins.rst

```diff
@@ -70,22 +70,21 @@ Model Fitting
 .. image:: ../img/model_fitting_components.png
 
 Astropy models can be fit to a spectrum via the Model Fitting plugin.
-Model components are selected via the :guilabel:`Model` pulldown menu.
-Each component should be given a :guilabel:`Model ID`.
-The :guilabel:`Add Model` button adds a Model Parameters block.
+Model components are selected via the :guilabel:`Model Component` pulldown menu.
+The :guilabel:`Add Component` button adds a Model Components block.
 
 Model Parameters are automatically initialized with a guess.
 These starting values can be edited by the user.
-They may also be fixed by selecting the :guilabel:`Fixed?` checkbox,
+They may also be fixed by selecting the checkbox,
 so that they are not fit or changed by the model fitting.
 
 A mathematical expression must be entered into the
-:guilabel:`Model Equation Editor` to specify the mathematical
+:guilabel:`Equation Editor` to specify the mathematical
 combination of models.
 This is also necessary even if there is only one model component.
-The model components are specified by their :guilabel:`Model ID`.
+The model components are specified by their label.
 For example, add together Constant and Gaussian models with
-model IDs 'C' and 'G1' by entering the Model Equation 'C+G1'.
+model labels 'C' and 'G1' by entering the Model Equation 'C+G1'.
```

Comment:
Adding the models together is the automatically populated default now, right? I would change this sentence to something like "By default, all models are added together, and you can edit the default Model Equation if desired. For example, the default Model Equation for Constant and Gaussian models with model labels 'C' and 'G1' is 'C+G1', which you could change to 'C/G1' if you wanted to divide the models instead".

Reply:
That's a good idea.  I modified what you suggested slightly because I'm not sure division will always work with units 😬 .  Let me know what you think.

"The model components are specified by their labels and the equation defaults to the sum of all created components, but can be modified to exclude some of components without needing to delete them entirely
or to change to subtraction, for example."

Reply:
Hmm...I actually think we should call out the valid operators explicitly (I think just  addition, subtraction, multiplication and division, pretty sure we don't support exponents and such here), and maybe add text saying that the models must yield results with compatible units to be combined?

Reply:
I created a follow-up ticket [:cat:](https://jira.stsci.edu/browse/JDAT-2402) to investigate and validate exactly what operators we support and then update this part of the docs as necessary - I think that anything other than addition/subtraction will likely raise an error right now, but there might be cases I'm not considering so I don't want to put that in writing quite yet.  Any improvements to the language that you would suggest in the meantime?

Reply:
This is fine for now then, assuming we actually get to that ticket in the near future.